tests/regression/tests/REQUEST-920-PROTOCOL-ENFORCEMENT/920120.yaml

```diff
@@ -839,3 +839,95 @@ tests:
               -----------------------------265001916915724--
           output:
             no_log_contains: id "920120"
+  - test_title: 920120-36
+    desc: "Attempted multipart/form-data bypass (920120). Still match ; that aren't part of an HTML entity"
+    stages:
+      - stage:
+          input:
+            dest_addr: 127.0.0.1
+            headers:
+              User-Agent: "ModSecurity CRS 3 Tests"
+              Host: "localhost"
+              Content-Type: multipart/form-data; boundary=---------------------------265001916915724
+              Accept: "text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5"
+            method: POST
+            port: 80
+            uri: /
+            data: |
+              -----------------------------265001916915724
+              Content-Disposition: form-data; name="&amp;fi;zzz"; filename="test"
+              Content-Type: application/octet-stream
+
+              helloworld
+              -----------------------------265001916915724--
+          output:
+            log_contains: id "920120"
+  - test_title: 920120-37
+    desc: "Attempted multipart/form-data bypass (920120). Still match ; that aren't part of an HTML entity."
+    stages:
+      - stage:
+          input:
+            dest_addr: 127.0.0.1
+            headers:
+              User-Agent: "ModSecurity CRS 3 Tests"
+              Host: "localhost"
+              Content-Type: multipart/form-data; boundary=---------------------------265001916915724
+              Accept: "text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5"
+            method: POST
+            port: 80
+            uri: /
+            data: |
+              -----------------------------265001916915724
+              Content-Disposition: form-data; name="test"; filename="&amp;fi;zzz"
+              Content-Type: application/octet-stream
+
+              helloworld
+              -----------------------------265001916915724--
+          output:
+            log_contains: id "920120"
+  - test_title: 920120-38
+    desc: "Attempted multipart/form-data bypass (920120). Still match ; that aren't part of an HTML entity."
```

Comment:
This looks like a positive test, right? Probably comment stayed because of copy&paste

Reply:
Yes, you're absolutely right. I've fixed the comments.